.dev/sass/components/_menus.scss

```diff
@@ -315,6 +315,60 @@ body.no-max-width .main-navigation {
 	}
 }
 
+.navigation.pagination {
+	clear: both;
+	margin: 0 0 1.5em;
+
+	.nav-links {
+		text-align: center;
+
+		.page-numbers {
+			display: inline-block;
+			border: none;
+			border-radius: 3px;
+			background-color: $color__background-body;
+			line-height: 1;
+			margin: 0 0.25em;
+			padding: 0.5em 0.75em;
+			white-space: nowrap;
+
+			&.dots {
+				background: none;
+			}
+
+			&.current {
+				color: $color__background-body;
```

Comment:
This should be addressed in the PR on primer https://github.com/godaddy/wp-primer-theme/pull/172